pages/common/zlib-flate.md

```diff
@@ -0,0 +1,17 @@
+# zlib-flate
+
+> Raw zlib compression and decompression program.
+> The zlib-flate program is part of the qpdf package.
+> More information: <https://manned.org/zlib-flate>.
+
+- Compress a file:
+
+`zlib-flate -compress < {{path/to/input_file}} > {{path/to/compressed.zlib}}`
+
+- Uncompress a file:
+
+`zlib-flate -uncompress < {{path/to/compressed.zlib}} > {{path/to/output_file}}`
+
+- Compress a file with a specified compression level [0-9]:
```

Comment:
Based on `gzip` page
```suggestion
- Compress a file with a specified compression level. 1=Fastest (Worst), 9=Slowest (Best):
```

Reply:
I did some tests to check about the level 0 (which apparently means "no compression at all") and turns out using the levels (besides 0) didn't change the size of the resulting file... I don't have any VMs right now, but would be nice if someone could test this on another machine to verify if the same thing occurs

Reply:
Yeah, I checked `0` is just save.
```suggestion
- Compress a file with a specified compression level. 0=Fastest (Worst), 9=Slowest (Best):
```